js-advanced-unit-5/vending-machines/money-functions.js

```diff
@@ -0,0 +1,50 @@
+const formatCurrency = amount => {
+  if (isNaN(amount)) {
+    return "Not a Number"
+  }
+
+  const amountRoundedAndFixedToTwo = (
```

Comment:
I would use a different variable name

```suggestion
  const formattedAmount = (
```